comms/lrzsz/Portfile

```
@@ -4,7 +4,7 @@ PortSystem          1.0
 
 name                lrzsz
 version             0.12.20
-revision            4
+revision            5
```

There's no need to increase the revision. Consult [the guide](https://guide.macports.org/#reference.keywords).

```suggestion
revision            4
```

---

Compile flags change for configure, which I think the guide suggests should increase the revision as it's similar to compile flag change?